src/components/Overlays/Popover.test.tsx

```
@@ -0,0 +1,33 @@
+
+vi.mock('@konturio/default-icons', () => ({
+  Close16: () => <svg data-testid="icon" />,
+}));
+import { Popover, PopoverTrigger, PopoverContent, PopoverClose } from './Popover';
+
+describe('Popover component', () => {
+  test('opens and closes via trigger', () => {
+    const { getByText, queryByText } = render(
+      <Popover initialOpen={false}>
+        <PopoverTrigger>
+          <button>open</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)